### DISCUSSION TITLE
Azure Postgres - Single Server - Backupy

### DISCUSSION THREAD

**Discussion by G-KROL** [AzureCommunityPL/group#341] [Pomoc]
Cześć,

jak podchodzicie do backupów w przypadku PostgreSQL? Teoretycznie istnieją backup z retencja do 35 dni i nawet georedundancja, ale sa one zarządzane przez Microsoft. Ostatnio przy tworzeniu pipeline  do testowania backupów, mocno się zdziwiłem że można zrobić tylko restora tylko z instniejącej bazy, co po doświadczenia z AWS jest raczej mało przydatne i bardzo ograniczające. Co więcej niby można miec 35 dni retencji, ale gdy ktos usunię baze ma się tylko 5 dni i to nadal nie jest do końca pewne. Znacie jakieś dobre praktyki, jak do tego dobrze podejść?

**Comment by piotrgwiazda** [comment]
Na razie pg_dump.
Long term backup dla PostgreSQL są w preview
https://azure.microsoft.com/en-us/blog/azure-backup-for-azure-postgresql-long-term-retention-in-preview/

Generalnie przy aktualizacji postgresa też się przyda, bo się nie da
aktualizować "w miejscu".

Zarządzanie Azure SQL jest daleko bardziej rozwinięte niż PostgreSQL w
Azure. Ale może wraz z Flexible Server doczekamy się.
https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/overview





pon., 20 wrz 2021, 11:37 użytkownik G-KROL ***@***.***>
napisał:
